test/unit/mdl-ripple/mdl-ripple.test.js

```diff
@@ -51,6 +51,13 @@ test('attachTo overrides unbounded data attr when explicitly specified', t => {
   t.end();
 });
 
+test('createAdapter() returns the same adapter used by default for the ripple', t => {
+  const root = bel`<div></div>`;
```

Comment:
oh yeah. I came across it via [yo-yo](https://www.npmjs.com/package/yo-yo), also really nice.